stackage-builder/configuration.nix

```diff
@@ -89,6 +89,8 @@
   # Enable the OpenSSH daemon.
   # services.openssh.enable = true;
 
+  services.fwupd.enable = true;
+
```

Comment:
Actually, I believe this file is unused and can just be deleted. In fact, `stackage-builder/nixos-modules` is good, but everything else under `stackage-builder` is unused after the migration to opencape!